templates/password-enhancements-error-messages.html.twig

```diff
@@ -0,0 +1,19 @@
+{#
+/**
+ * @file
+ * Template for the password validation errors.
+ *
+ * Available variables:
+ *   - error_message (string): The error message itself.
+ *   - messages (string[]): The error messages from each failed validation.
+ */
+#}
+
+<div>
+  <div class="password-error-message">{{ error_message }}</div>
+  <ul>
+    {% for message in messages %}
```

Comment:
Another cool thing would be if you introduced [Twig blocks](https://twig.symfony.com/doc/2.x/tags/block.html): `{% block messages %}{% endblock %}` to your templates.

Similarly to the above solution a themer can override just the parts they need to override instead of changing everything in the original template. Let's say you have this template:
```
{% block messages %}
<div>
  {% block error-message %}
    <div class="password-error-message">{{ error_message }}</div>
  {% endblock %}
  <ul>
    {% for message in messages %}
      <li>{{ message }}</li>
    {% endfor %}
  </ul>
</div>
{% endblock%}
```

A themer can simply extend and overwrite only the part they need:

```
{% extends '@password_enhancements/password-enhancements-error-messages.html.twig %}
{% block error-message %}
      <div class="password-error-message">{{ error_message }}<span class="error-message-icon"></span></div>
{% endblock %}
```